src/index.tsx

```diff
@@ -13,6 +13,21 @@ const client = new ApolloClient({
   cache: new InMemoryCache(),
 });
 
+// create an app component
```

Comment:
Despite the fact that we may need this code in the future, I see no reason to keep it while it is not currently being used.

You can safely remove it for now

Reply:
Solved.